addons/web/static/tests/views/list_tests.js

```diff
@@ -4194,6 +4194,50 @@ QUnit.module('Views', {
 
         list.destroy();
     });
+
+    QUnit.test('readonly boolean in editable list is readonly', function (assert) {
+        assert.expect(6);
+
+        var list = createView({
+            View: ListView,
+            model: 'foo',
+            data: this.data,
+            arch: '<tree editable="bottom">'
+                      + '<field name="foo"/>'
+                      + '<field name="bar" modifiers="{&quot;readonly&quot;: [[&quot;foo&quot;, &quot;!=&quot;, &quot;yop&quot;]]}"/>'
+                  +'</tree>',
```

Comment:
'modifiers' is a key added by the view processing. You should define 'attrs' instead (and i prefer \' than $quot;).
Also, the '+' should be at the end of the previous line.

Reply:
ok all done

and I don't know why I thought of using modifiers instead of `attrs` but since modifiers is JSON the `'` were not possible.